content/en/docs/tasks/traffic-management/istioctl-describe/index.md

```diff
@@ -0,0 +1,241 @@
+
+First, let us [deploy Bookinfo](/docs/examples/bookinfo/).
+
+{{< text bash >}}
+$ kubectl apply -f samples/bookinfo/platform/kube/bookinfo.yaml
+$ kubectl apply -f samples/bookinfo/networking/bookinfo-gateway.yaml
+$ export INGRESS_HOST=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+$ export INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="http2")].port}')
+$ export GATEWAY_URL=$INGRESS_HOST:$INGRESS_PORT
+$ echo Bookinfo at $GATEWAY_URL/productpage
+{{< /text >}}
+
+Let's describe a pod:
+
+{{< text bash >}}
+$ export RATINGS_POD=$(kubectl get pod -l app=ratings -o jsonpath='{.items[0].metadata.name}')
+$ istioctl experimental describe pod $RATINGS_POD
+{{< /text >}}
+
+The output tells us which containers the pod exposes, the Istio protocol for the microservice on port 9080, and the mutual TLS settings for the pod.
+
+{{< text plain >}}
+Pod: ratings-v1-f745cf57b-qrxl2
+   Pod Ports: 9080 (ratings), 15090 (istio-proxy)
+--------------------
+Service: ratings
+   Port: http 9080/HTTP
+Pilot reports that pod enforces HTTP/mTLS and clients speak HTTP
+{{< /text >}}
+
+## Destination Rules
+
+Next we apply the destination rules suggested by the documentation.  I used mutual TLS,
+so I must apply `destination-rule-all-mtls.yaml`:
+
+{{< text bash >}}
+$ kubectl apply -f samples/bookinfo/networking/destination-rule-all-mtls.yaml
+$ istioctl x describe pod $RATINGS_POD
+{{< /text >}}
+
+Applying `destination-rule-all-mtls.yaml` created four destination rules: `details`, `productpage`, `ratings`, and `reviews`.
+
+{{< text bash >}}
+$ istioctl x describe pod $RATINGS_POD
+{{< /text >}}
+
+Now `istioctl x describe` shows additional output:
+
+{{< text plain >}}
+Pod: ratings-v1-f745cf57b-qrxl2
+   Pod Ports: 9080 (ratings), 15090 (istio-proxy)
+--------------------
+Service: ratings
+   Port: http 9080/HTTP
+DestinationRule: ratings for "ratings"
+   Matching subsets: v1
+      (Non-matching subsets v2,v2-mysql,v2-mysql-vm)
+   Traffic Policy TLS Mode: ISTIO_MUTUAL
+Pilot reports that pod enforces HTTP/mTLS and clients speak mTLS
+{{< /text >}}
+
+The destination rule now appears in the output.  This tells us that the `ratings` destination rule is present, and that it defines the subset `v1` which matches this pod.
+Clients talking to the ratings microservice will use mutual TLS.
+
+## Virtual Services
+
+Now I will follow the Bookinfo example to [Request Routing](/docs/tasks/traffic-management/request-routing/) and define some virtual services:
+
+{{< text bash >}}
+$ kubectl apply -f samples/bookinfo/networking/virtual-service-all-v1.yaml
+{{< /text >}}
+
+After applying this rule I "describe" the _reviews-v1_ pod:
+
+{{< text bash >}}
+$ export REVIEWS_V1_POD=$(kubectl get pod -l app=reviews,version=v1 -o jsonpath='{.items[0].metadata.name}')
+$ istioctl x describe pod $REVIEWS_V1_POD
```

Comment:
should we use istioctl x in other places too to be consistent?  you had some using experimental earlier.